src/app/elastic/search.py

```diff
@@ -30,3 +35,16 @@ def execute(self, ignore_cache=False) -> Response:
             pass
 
         return response
+
+
+def add_base_filters(search: Search) -> Search:
+    for entry in base_filter:
+        search = search.filter(entry)
+    return search
+
+
+base_filter = [
```

Comment:
put it inside the `Search` class as (private?) static variable?

Reply:
Done